pkg/config/config.go

```diff
@@ -1,55 +1,17 @@
 package config
 
-import (
-	"fmt"
-	"os"
-	"time"
-
-	"github.com/aws/eks-anywhere/pkg/logger"
-)
-
 const (
-	EksaGitPassphraseTokenEnv   = "EKSA_GIT_SSH_KEY_PASSPHRASE"
-	EksaGitPrivateKeyTokenEnv   = "EKSA_GIT_PRIVATE_KEY"
-	EksaGitKnownHostsFileEnv    = "EKSA_GIT_KNOWN_HOSTS"
-	SshKnownHostsEnv            = "SSH_KNOWN_HOSTS"
-	EksaReplicasReadyTimeoutEnv = "EKSA_REPLICAS_READY_TIMEOUT"
-	ExternalEtcdTimeoutEnv      = "EKSA_EXTERNAL_ETCD_TIMEOUT"
-	EksaAccessKeyIdEnv          = "EKSA_AWS_ACCESS_KEY_ID"
-	EksaSecretAcessKeyEnv       = "EKSA_AWS_SECRET_ACCESS_KEY"
-	EksaRegionEnv               = "EKSA_AWS_REGION"
+	EksaGitPassphraseTokenEnv = "EKSA_GIT_SSH_KEY_PASSPHRASE"
+	EksaGitPrivateKeyTokenEnv = "EKSA_GIT_PRIVATE_KEY"
+	EksaGitKnownHostsFileEnv  = "EKSA_GIT_KNOWN_HOSTS"
+	SshKnownHostsEnv          = "SSH_KNOWN_HOSTS"
+	EksaAccessKeyIdEnv        = "EKSA_AWS_ACCESS_KEY_ID"
+	EksaSecretAcessKeyEnv     = "EKSA_AWS_SECRET_ACCESS_KEY"
+	EksaRegionEnv             = "EKSA_AWS_REGION"
 )
 
 type CliConfig struct {
 	GitSshKeyPassphrase string
 	GitPrivateKeyFile   string
 	GitKnownHostsFile   string
-	MaxWaitPerMachine   time.Duration
-}
-
-const (
-	DefaultMaxWaitPerMachine = 10 * time.Minute
-	DefaultEtcdWaitStr       = "60m"
-)
-
-func GetMaxWaitPerMachine() time.Duration {
-	if env, found := os.LookupEnv(EksaReplicasReadyTimeoutEnv); found {
```

Comment:
We should probably let the team know we are removing support for these env vars so they start using the flags if needed.

Reply:
Yes I did communicate in CLI standup that I would be moving timeout env vars to cli options. I will re-communicate it to the team.